console/src/dataSources/services/mysql/sqlUtils.ts

```diff
@@ -553,5 +554,14 @@ ORDER BY
 
 export const primaryKeysInfoSql = (options: { schemas: string[] }) =>
   getKeysSql('PRIMARY KEY', options);
+
 export const uniqueKeysSql = (options: { schemas: string[] }) =>
   getKeysSql('UNIQUE', options);
+
+const schemaListSQL = `SELECT schema_name FROM information_schema.schemata WHERE
+schema_name NOT IN (SELECT schema_name FROM information_schema.schemata
+WHERE schema_name = 'information_schema' OR schema_name = 'mysql' OR schema_name = 'sys'
+OR schema_name = 'performance_schema') ORDER BY schema_name ASC;`;
```

Comment:
The original rql query had the condition in order to place the nulls towards the end. Is that still necessary?